docs/technical-doc/architecture.rst

```diff
@@ -0,0 +1,340 @@
+``target_address``               The address of the targeted smart contract
+``target_fun``                   The function that is called              
+``params``                       The parameters of the function call              
+===============================  =========================================================
+
+Block
+*****
+
+A block is a data structure built by nodes and its function it to aggregate several operations. As explained above, for each new slot that becomes active, a particular node in the network is elected in a deterministic way with the task of creating the block that will be stored in that slot (more about this in the description of the Selector Module below). A block from a given thread can only contain operations originating from a creator_public_key whose hash's five first bits designate the corresponding thread, thus implicitly avoiding collisions in operations integrated into parallel threads.
+
+The content of a block is as follows:
+
+===============================  =========================================================
+**Block header**       
+------------------------------------------------------------------------------------------ 
+``slot``                         A description of the block slot, defined by a couple (period, thread) that 
+                                 uniquely identify it
+``creator_public_key``           The public key of the block creator (64 bytes)           
+``parents``                      A list of the 32 parents of the block, one parent per thread (parent blocks are 
+                                 identified by the block hash)        
+``endorsements``                 A list of the 9 endorsements for the block (more about endorsements below)
+``operations_hash``              A hash of all the operations included in the block (=hash of the block body below)
+``signature``                    signature of all the above with the private key of    
+                                 the block creator                               
+**Block body**       
+------------------------------------------------------------------------------------------ 
+``operations``                   The list of all operations included in the block                         
+===============================  =========================================================
+
+Endorsements are optional inclusion in the block, but their inclusion is incentivized for block creators. They are validations of the validity of the parent block on the thread of the block, done by other nodes that have also been deterministically selected via the proof of stake probability distribution (see below). A comprehensive description of endorsements can be found `here <https://github.com/massalabs/massa/blob/main/docs/technical-doc/Endorsements.rst>`_, so we will not go further into details in the context of this introduction.
+
+Architecture
```

Comment:
I'll leave it for a bit later, because the introduction is really an intro to the architecture for now, but it would indeed be interesting to turn it as a general massa tech intro (in that case, some info is missing in my opinion).